DONT-MERGE kodi: 16.1 -> 17.0

edwtjo opened the pull request:
###### Motivation for this change

Test uppcoming Kodi, [changelog](http://kodi.wiki/view/Kodi_v17_%28Krypton%29_changelog). @cpages reading the changelog it seems that pvr-hts might need special care, I don't use that feature.
###### Things done
- [ ] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [ ] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

edwtjo committed:
kodi: 16.1 -> 17.0

cpages commented:
Not using it right now, but I'll try to have a look anyway. Pinging @simonvandel who fixed it last time.

cpages committed:
kodiPlugins.pvr-hts: update to 3.4.4 for kodi17

Updated pvr-hts plugin to latest version to work with kodi 17. Had to
update libcec as well, to pick the name change.

edwtjo committed:
kodiPlugins: updates

  * Added exodus in favour of genesis, same developer and old plugin is dead
  * Added gamepad(controllers) inputs as enableControllers
  * Updated SVTPlay, HyperLauncher
  * Removed SALTS, terrible quality plugin

edwtjo commented:
cherry-picked a70b4f7b3896b082c8547797f7e676de49d72ea3 d211d7c44009df5fd57b231cac183250b56a6cf5 62dc6830967ceb2d18630503ae9fecbabfc5ffa0
